docs/research/rhubarb_test/src/index.ts

```diff
@@ -0,0 +1,41 @@
+import { exec } from 'node:child_process'
+import { stdout, chdir, cwd } from 'node:process';
+
+console.log("Start Rhubard....")
+
+console.log(`Starting directory: $${cwd()}`)
+
+cwd()
+
+try {
+    chdir('../rhubarb')
+    console.log(`New directory: ${cwd()}`)
+} catch(err) {
+    console.log(`Error message: ${err}`)
+}
+
+exec("chmod +x ./rhubarb.exe", (error,stout) => {
+    if (error) {
+        throw error } else {
+            console.log(stdout)
+        } 
+})
+
+cwd()
+
+exec('"./rhubarb.exe" -o output.txt amber.wav', (error,stdout,stderr) => {
```

Comment:
when we integrate it further later on, we'll need to check for the language, as we'll need to change the recognizer depending on whether it's English or non-English